spec/rails_helper.rb

```diff
@@ -61,5 +61,8 @@
     with.test_framework :rspec
     with.library :rails
   end
+end
 
+RSpec.configure do |config|
+  config.include Rails.application.routes.url_helpers
```

Comment:
I was running into an issue referencing root_path and thought this may help but found it was related to something different.

I kept it in the code because I think referencing paths via the helpers will make the specs more adaptable in the case that I change these routes in the future.